internal/pkg/services/argus/utils/utils_test.go

```diff
@@ -220,3 +349,87 @@ func TestValidatePlanId(t *testing.T) {
 		})
 	}
 }
+
+func TestMapToUpdateScrapeConfigPayload(t *testing.T) {
+	tests := []struct {
+		description     string
+		resp            *argus.GetScrapeConfigResponse
+		expectedPayload *argus.UpdateScrapeConfigPayload
+		isValid         bool
+	}{
+		{
+			description:     "base case",
+			resp:            fixtureGetScrapeConfigResponse(),
+			expectedPayload: fixtureUpdateScrapeConfigPayload(),
+			isValid:         true,
+		},
+		{
+			description: "nil response",
+			resp:        nil,
+			isValid:     false,
+		},
+		{
+			description: "nil data",
+			resp: &argus.GetScrapeConfigResponse{
+				Data: nil,
+			},
+			isValid: false,
+		},
+		{
+			description: "empty data",
+			resp: &argus.GetScrapeConfigResponse{
+				Data: &argus.Job{},
+			},
+			expectedPayload: &argus.UpdateScrapeConfigPayload{},
+			isValid:         true,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.description, func(t *testing.T) {
+			payload, err := MapToUpdateScrapeConfigPayload(tt.resp)
+
+			if tt.isValid && err != nil {
+				t.Errorf("failed on valid input")
+			}
+			if !tt.isValid && err == nil {
+				t.Errorf("did not fail on invalid input")
+			}
+			if !tt.isValid {
+				return
+			}
+
+			diff := cmp.Diff(*payload, *tt.expectedPayload,
+				cmp.AllowUnexported(*tt.expectedPayload),
+			)
+			if diff != "" {
+				t.Fatalf("Data does not match: %s", diff)
+			}
+		})
+	}
+}
+
+func TestGetDefaultCreateScrapeConfigPayload(t *testing.T) {
```

Comment:
This test is not testing much and will add maintenance effort because if the defaults are updated then they also need to be updated here. The function is like a getter, I would remove this test.

Probably what make more sense is to make the `DefaultCreateScrapeConfig` a const instead of a function